openpype/hosts/maya/startup/userSetup.py

```diff
@@ -13,24 +12,41 @@
 
 print("Starting OpenPype usersetup...")
 
+project_settings = get_project_settings(os.environ['AVALON_PROJECT'])
+
+# Loading plugins explicitly.
+explicit_plugins_loading = project_settings["maya"]["explicit_plugins_loading"]
+if explicit_plugins_loading["enabled"]:
+    def _explicit_load_plugins():
+        for plugin in explicit_plugins_loading["plugins_to_load"]:
+            if plugin["enabled"]:
+                print("Loading plug-in: " + plugin["name"])
+                try:
+                    cmds.loadPlugin(plugin["name"], quiet=True)
+                except RuntimeError as e:
+                    print(e)
+
+    # We need to load plugins deferred as loading them directly does not work
+    # correctly due to Maya's initialization.
```

Comment:
Can we make this more explicit. What doesn't work? Why doesn't it work? Doesn't it work due to a specific plug-in trying to access the UI even though it doesn't exist yet? Doesn't it work because Maya crashes?

If we know what happens or what fails it means we can come back here in 5 years time and read the comment to test whether it's still a potential issue instead of just relying on it 'not working'.

What do you think?

Reply:
Sorry, I dont know why its not working, but I'm not sure spending time on figure that out, is time well spent.

Reply:
How did you noticed it didn't work? What happened? It didn't load the plug-ins? It failed to start? Even hinting at what actually occurred makes more sense to me. It's just that by reading the comment I currently have no idea what to potentially look out for.

Reply:
```suggestion
    # We need to load plugins deferred as loading them directly does not work
    # correctly due to Maya's initialization. Maya launch crashes when for
    # example enabling the plugin "gpuCache".
```

Cool, have added some more information.